### PR TITLE
New version: DuckStudio.GitHubLabelsManager version 1.6

### LINKED ISSUE #195955: [Update Request]: DuckStudio.GitHubLabelsManager version 1.6
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

DuckStudio.GitHubLabelsManager

### Package Version

1.6

### Please describe the changes you would like to see

https://github.com/DuckDuckStudio/GitHub-Labels-Manager/releases/tag/v1.6

### PR DESCRIPTION
**Pull request by DuckDuckStudio**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue? - Resolves #195955

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/195954)

### CODE REVIEW

**Review by Luna-Grace — APPROVED** [review]

File: manifests/d/DuckStudio/GitHubLabelsManager/1.6/DuckStudio.GitHubLabelsManager.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  Architecture: arm
+  InstallerUrl: https://github.com/DuckDuckStudio/GitHub-Labels-Manager/releases/download/v1.6/GitHub_Labels_Manager_Setup_v1.6-EN.exe
+  InstallerSha256: B2FD49024DCE40586B546F07880C94026E14AA37A4818507BFB5EF19A48DCBC9
+- InstallerLocale: en-US
+  Architecture: arm64
+  InstallerUrl: https://github.com/DuckDuckStudio/GitHub-Labels-Manager/releases/download/v1.6/GitHub_Labels_Manager_Setup_v1.6-EN.exe
+  InstallerSha256: B2FD49024DCE40586B546F07880C94026E14AA37A4818507BFB5EF19A48DCBC9
+
+ManifestType: installer
+ManifestVersion: 1.7.0
```
Comment (Luna-Grace) [inline manifests/d/DuckStudio/GitHubLabelsManager/1.6/DuckStudio.GitHubLabelsManager.installer.yaml:56]:
```suggestion
ManifestVersion: 1.6.0
```

File: manifests/d/DuckStudio/GitHubLabelsManager/1.6/DuckStudio.GitHubLabelsManager.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+  **Full Changelog**: https://github.com/DuckDuckStudio/GitHub-Labels-Manager/compare/v1.5...v1.6
+ReleaseNotesUrl: https://github.com/DuckDuckStudio/GitHub-Labels-Manager/releases/tag/v1.6
+# PurchaseUrl:
+# InstallationNotes:
+Documentations: 
+- DocumentLabel: README
+  DocumentUrl: https://github.com/DuckDuckStudio/GitHub-Labels-Manager/blob/main/other-languages/en_US/README.md
+ManifestType: locale
+ManifestVersion: 1.7.0
```
Comment (Luna-Grace) [inline manifests/d/DuckStudio/GitHubLabelsManager/1.6/DuckStudio.GitHubLabelsManager.locale.en-US.yaml:52]:
```suggestion
ManifestVersion: 1.6.0
```

File: manifests/d/DuckStudio/GitHubLabelsManager/1.6/DuckStudio.GitHubLabelsManager.locale.zh-CN.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+  **Full Changelog**: https://github.com/DuckDuckStudio/GitHub-Labels-Manager/compare/v1.5...v1.6
+ReleaseNotesUrl: https://github.com/DuckDuckStudio/GitHub-Labels-Manager/releases/tag/v1.6
+# PurchaseUrl:
+# InstallationNotes:
+Documentations: 
+- DocumentLabel: README
+  DocumentUrl: https://github.com/DuckDuckStudio/GitHub-Labels-Manager/blob/main/README.md
+ManifestType: defaultLocale
+ManifestVersion: 1.7.0
```
Comment (Luna-Grace) [inline manifests/d/DuckStudio/GitHubLabelsManager/1.6/DuckStudio.GitHubLabelsManager.locale.zh-CN.yaml:53]:
```suggestion
ManifestVersion: 1.6.0
```

File: manifests/d/DuckStudio/GitHubLabelsManager/1.6/DuckStudio.GitHubLabelsManager.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=NVS1.CRLF.5-1-22621-3958.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: DuckStudio.GitHubLabelsManager
+PackageVersion: "1.6"
+DefaultLocale: zh-CN
+ManifestType: version
+ManifestVersion: 1.7.0
```
Comment (Luna-Grace) [inline manifests/d/DuckStudio/GitHubLabelsManager/1.6/DuckStudio.GitHubLabelsManager.yaml:8]:
```suggestion
ManifestVersion: 1.6.0
```

**Review by Luna-Grace — COMMENTED** [review]
```
> winget validate --manifest D:\...\winget-pkgs\manifests\d\DuckStudio\GitHubLabelsManager\1.6
清单验证成功。
```

Why is it flaged as https://github.com/microsoft/winget-pkgs/labels/Manifest-Version-Error ?
If the manifest version is not supported, an error message should be displayed when validating the manifest, such as:

```
> winget validate --manifest D:\...\winget-pkgs\manifests\d\DuckStudio\GitHubLabelsManager\1.6
清单验证失败。
Unsupported ManifestVersion: 2.7.0
```

**Comment by DuckDuckStudio** [comment]
> === manifests\d\DuckStudio\GitHubLabelsManager\1.6 ===
> Missing Properties value based on version v1.4: 
> Switches

- See https://github.com/microsoft/winget-pkgs/pull/195394#issuecomment-2508954783

**Review by Luna-Grace — APPROVED** [review]

**Review by fjwxzde — APPROVED** [review]

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> No errors to post.

(Automated response - build 899.)

**Comment by DuckDuckStudio** [comment]
"IDES"...? again?

**Comment by DuckDuckStudio** [comment]
> === manifests\d\DuckStudio\GitHubLabelsManager\1.6 ===
> Missing Properties value based on version v1.4: 
> Switches

- See https://github.com/microsoft/winget-pkgs/pull/195394#issuecomment-2508954783

**Review by stephengillie — APPROVED** [review]

**Pull request merged**